pkg/cloudprovider/aws/block_store.go

```diff
@@ -95,14 +97,23 @@ func (b *blockStore) CreateVolumeFromSnapshot(snapshotID, volumeType, volumeAZ s
 		return "", errors.Errorf("expected 1 snapshot from DescribeSnapshots for %s, got %v", snapshotID, count)
 	}
 
+	tags := make(map[string]string)
```

Comment:
any reason to have this section of code outside of the `getTagsForCluster` func? I'd probably opt to move this inside